ci/release/run.sh

```diff
@@ -12,5 +12,5 @@ npx --yes \
   -p "@semantic-release/github" \
   -p "@semantic-release/exec" \
   -p "@semantic-release/git" \
-  -p "conventional-changelog-conventionalcommits@7.0.2" \
+  -p "conventional-changelog-conventionalcommits" \
```

Comment:
```suggestion
  -p "conventional-changelog-conventionalcommits@8.0.0" \
```

If we do this and also pin semantic-release itself to `"semantic-release@24.0.0"` that should prevent this breaking in the future (we just hit this in Ibis, which is where we copied this release process from)